AwsCryptographyPrimitives/runtimes/rust/Cargo.toml

```diff
@@ -13,6 +13,7 @@ aws-lc-sys = "0.22.0"
 aws-smithy-runtime-api = "1.7.3"
 aws-smithy-types = "1.2.9"
 chrono = "0.4.38"
+cpu-time = "1.0.0"
```

Comment:
(Curious) I'm surprised you need to include this in every cargo.toml file -- is this not transitively included from StandardLibrary?

Reply:
Nope. In Dafny-Rust there is no transitive. Everything is built as one giant package, and each needs its own separate Cargo.toml.